corehq/apps/hqwebapp/static/app_manager/less/content.less

```diff
@@ -275,7 +275,7 @@
 .case-tile-preview {
     display: grid;
     grid-template-columns: repeat(12, 1fr) min-content 5px;
-    grid-template-rows: repeat(3, auto) min-content 5px;
```

Comment:
does this affect the other template previews?

Reply:
No, the other template previews look the same before and after